src/js/_enqueues/admin/user-profile.js

```diff
@@ -332,6 +340,75 @@
 		}
 	}
 
+	/**
+	 * Binds Caps Lock detection to a given password input field.
+	 *
+	 * @param {jQuery} $input The password input field.
+	 */
+	function bindCapsLockWarning( $input ) {
+		var $capsWarning = $( '#caps-warning' ),
+			capsLockOn   = false;
+
+		$input.on( 'keydown', function( e ) {
+			var ev = e.originalEvent || e;
+
+			// Skip CapsLock key itself.
+			if ( ev.key === 'CapsLock' ) {
+				if (capsLockOn) {
```

Comment:
JS coding standards for whitespace should be followed.

```suggestion
				if ( capsLockOn ) {
```